src/Microsoft.DotNet.Interactive/Events/ValueNamesProduced.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.Events
+{
+    public class ValueNamesProduced : KernelEvent
+    {
+        public IReadOnlyCollection<string> ValueNames { get; }
```

Comment:
This is going to break when we add supported MIME types for each value name. Let's discuss whether we should do that now.

Reply:
done